docs/nftfair/contracts/vault/base.md

```diff
@@ -97,51 +105,82 @@ The owner can invoke this method to withdraw remaining NFTs from one or more spe
 | Categories | `uint256[]` | One or more categories |
 | Recipient | `address` | The recipient address |
 
-## `claimed`
+### `claimed`
 
 ```solidity
 function claimed(address _user, uint256 _category) public view override returns (uint256[] memory)
 ```
 
 Returns a list of token IDs that has been claimed by the `_user` from a specified `_category`
 
-## `getClaimableIds`
+### `getClaimableIds`
 
 ```solidity
 function getClaimableIds(uint256 _category) public view override returns (uint256[] memory)
 ```
 
 Returns a list of available token IDs from a specified `_category`.
 
-## `getCategoryPrice`
+### `getCategoryPrice`
 
 ```solidity
 function getCategoryPrice(uint256 _category) external view override returns (uint256)
 ```
 Returns the amount of `paymentToken` required to purchase a single NFT from the specified `_category`. Call the `paymentToken()` method to get the ERC20 address of the payment token.
 
-## `getCategoryNft`
+### `getCategoryNft`
 
 ```solidity
 function getCategoryNft(uint256 _category) external view override returns (address)
 ```
 Returns the address of the NFT contract from the specified `_category`.
 
-## `getCategoryNftIds`
+### `getCategoryNftIds`
 
 ```solidity
 function getCategoryNftIds(uint256 _category) external view returns (uint256[] memory)
 ```
 Returns the list of token IDs that has been added into this category. This list excludes token IDs that had been recovered by the owner.
 
-## `claimableCount`
+### `claimableCount`
 
 ```solidity
 function claimableCount(uint256 _category) external view returns (uint256)
 ```
 
 Return the number of available token IDs from a specified `_category`.
 
+ERC1155: This method returns the total token amount to claim, spanned across all token IDs that are available. For example, if the vault contains 1 of token ID: 1 and 2 of token ID: 2, this method returns 3, since there are three tokens available to claim.
+
+---
+## ERC 1155 Compatibility
+
+This section describes the methods that are only valid for `categpry.tokenType == ERC1155`.
+
+### `claimedCount`
+
+```solidity
+function claimedCount(address _user, uint256 _category) public view returns (uint256)
+```
+
+Returns the total number of NFTs that have been claimed by the _user from a specified _category
+
+### `getClaimableCountByIds`
```

Comment:
What's the difference of this 2 functions?

Reply:
`_claimedCount` is the sum of tokens that are claimed by a given `_user` in the specified `_category`.

`getClaimableCountByIds` is the list of amount of tokens that are available to claim corresponding to the list of `_tokenId` provided.

Reply:
oh sorry, I didn't notice the `claimed`, thanks